pkg/provider/provider_helpers.go

```diff
@@ -201,3 +167,46 @@ func GetAccessTokenWithRefreshToken(
 func envNameFieldDescription(description, envName string) string {
 	return fmt.Sprintf("%s Can also be sourced from the `%s` environment variable.", description, envName)
 }
+
+func handleStringField(d *schema.ResourceData, key string, field *string) error {
+	if v, ok := d.GetOk(key); ok {
+		*field = v.(string)
+	}
+	return nil
+}
+
+func handleBoolField(d *schema.ResourceData, key string, field *bool) error {
+	if v, ok := d.GetOk(key); ok {
+		*field = v.(bool)
+	}
+	return nil
+}
+
+func handleDurationAttribute(d *schema.ResourceData, key string, field *time.Duration) error {
```

Comment:
Include in function name that the value is calculated in seconds

Reply:
Done.